<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pivote_Parcial</class>
 <widget class="QWidget" name="Pivote_Parcial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vector b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matriz A</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_b">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>101</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="solve">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Solucionar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_A">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>256</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_setAb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>agregar Matriz A y vectos B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Soluci√≥n</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_resolut">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_solve">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
